lib/SPIRV/libSPIRV/SPIRVDecorate.h

```diff
@@ -102,6 +102,17 @@ class SPIRVDecorateGeneric : public SPIRVAnnotationGeneric {
     }
   }
 
+  llvm::Optional<ExtensionID> getRequiredExtension() const override {
+    switch (static_cast<size_t>(Dec)) {
+    case internal::BuiltInSubDeviceIDINTEL:
```

Comment:
I was surprised by this change. Don't we have any extensions that bring new decorations already?